keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/nuts/NUTS.java

```diff
@@ -31,6 +36,12 @@
     private static final int DEFAULT_ADAPT_COUNT = 1000;
     private static final double DEFAULT_TARGET_ACCEPTANCE_PROB = 0.65;
 
+    enum Metrics {
```

Comment:
This needs to be public, since in real world usage it will be referenced by calling code outside of this package.